src/Components/LocationForm/LocationForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import FormButton from "./FormButton";
```

Comment:
You can move (and rename) all the form-related components to the same folder of this component.
I suggest you use:
`/Components/LocationForm/Form.js` for this one, and `Button`, `CoordinatesInput`, `CoordinatesList`, and `Subtitle` (instead of `Legend`) for the children, all of them inside the `/Components/LocationForm` folder.

Reply:
Oh, forget, the components are already in the same folder, I think only the names could be simplified, removing the Form from them. I just keep my suggestion for the new names! :)

Reply:
Sure, better naming! Done!